Dockerfile

```diff
@@ -59,3 +87,13 @@ RUN mkdir ~/behat && \
         "behat/mink-extension:^2.2" \
         "behat/mink-goutte-driver:^1.2" \
         "drupal/drupal-extension:*"
+
+###########################
+# Finalize headless Chrome
+# Borrowed from https://github.com/GoogleChrome/puppeteer/docs/troubleshooting.md#running-puppeteer-in-docker
+###########################
+
+# Run everything after as non-privileged user.
+USER pptruser
```

Comment:
Some of the existing users of this docker image may assume the user name et. al; we should keep the existing unpriv user code above instead of changing to this new one. Just move the old code down here.